test/client/fixtures/sandbox/code-instrumentation/setters-test.js

```diff
@@ -622,3 +622,29 @@ test('setter returns a correct value (GH-907)', function () {
     checkReturnedValue('img', 'src', './path');
     checkReturnedValue('a', 'target', '_blank');
 });
+
+test('should not throw an error on setting the body.innerHtml when document.body equals null (GH-1172)', function () {
+    var iframe = document.createElement('iframe');
+
+    iframe.id = 'test' + Date.now();
+
+    document.body.appendChild(iframe);
+
+    iframe.contentDocument.write(
+        '<head>',
```

Comment:
The `processScript` function is called into the `iframe.contentDocument.write` function.
I removed setter from the html.